float64[42] Jacobian
float64[49] MassMatrix
float64[7] Coriolis
float64[7] Gravity
float64[7] q
float64[7] dq
float64[7] tau_J
float64[16] O_T_EE
float64[16] F_T_EE
float64[16] NE_T_EE
float64[16] EE_T_K
float64[6] O_F_ext_hat_K
float64[6] K_F_ext_hat_K 

